.github/workflows/be-prod-deploy-hotfix.yml

```
@@ -84,25 +88,28 @@ jobs:
           chmod +x ./gradlew
-          ./gradlew build
+          ./gradlew test
 
-      - name: Create artifact package
-        run: |
-          mkdir deploy-artifact
-          cp server/build/libs/*[!plain].jar deploy-artifact/ahmadda-server-prod.jar
-          cp server/codedeploy/appspec.yml deploy-artifact/
-          cp -r server/codedeploy/scripts/ deploy-artifact/
+      - name: Log in to Docker Hub
+        uses: docker/login-action@v2
```

actionlint (1.7.8)

the runner of "docker/login-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)